components/avatar/index.css

```diff
@@ -10,107 +10,128 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-avatar-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
+  --spectrum-avatar-focus-indicator-color: var(--spectrum-focus-indicator-color);
+
+  --spectrum-avatar-color-opacity-disabled: var(--spectrum-avatar-opacity-disabled);
+}
 
 .spectrum-Avatar--size50 {
-  @remapvars {
-    find: /--spectrum-avatar-size-50(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-50);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-50)
 }
 
 .spectrum-Avatar--size75 {
-  @remapvars {
-    find: /--spectrum-avatar-size-75(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-75);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-75)
 }
 
 .spectrum-Avatar--size100 {
-  @remapvars {
-    find: /--spectrum-avatar-size-100(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-100);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-100)
 }
 
 .spectrum-Avatar--size200 {
-  @remapvars {
-    find: /--spectrum-avatar-size-200(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-200);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-200)
 }
 
 .spectrum-Avatar--size300 {
-  @remapvars {
-    find: /--spectrum-avatar-size-300(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-300);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-300)
 }
 
 .spectrum-Avatar--size400 {
-  @remapvars {
-    find: /--spectrum-avatar-size-400(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-400);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-400)
 }
 
 .spectrum-Avatar--size500 {
-  @remapvars {
-    find: /--spectrum-avatar-size-500(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-500);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-500)
 }
 
 .spectrum-Avatar--size600 {
-  @remapvars {
-    find: /--spectrum-avatar-size-600(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
-  }
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-600);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-600)
 }
 
 .spectrum-Avatar--size700 {
-  @remapvars {
-    find: /--spectrum-avatar-size-700(.*)/;
-    filter: color;
-    replace: --spectrum-avatar$1;
+  --spectrum-avatar-inline-size: var(--spectrum-avatar-size-700);
+  --spectrum-avatar-block-size: var(--spectrum-avatar-size-700)
+}
+
+@media (forced-colors: active) {
+  .spectrum-Avatar {
+    --highcontrast-avatar-color-opacity-disabled: 1;
+    --highcontrast-avatar-focus-indicator-color: Highlight;
```

Comment:
James Nurthen and I discussed what our focused component border color and focus indicator border color should be in WHCM. The verdict is:

**Component border (if there is one):** Highlight
**Focus indicator border color:** CanvasText

Since Avatar does not have a border, only the focus indicator color needs to be changed to CanvasText.

It may also be necessary to add "forced-color-adjust: none;" to the focus indicator. Here is the format:

```
@media (forced-colors: active) {
  .spectrum-Avatar:not(.is-disabled) .spectrum-Avatar-link.focus-ring,
  .spectrum-Avatar:not(.is-disabled) .spectrum-Avatar-link:focus-visible {
    &::after {
      forced-color-adjust: none;
    }
  }
}
```

Reply:
It looks like we're ok without having to add the `forced-color-adjust: none;`

Here's what I'm seeing in AssitivLabs after updating to use `CanvasText`:

<img width="151" alt="Screen Shot 2023-01-10 at 5 01 35 PM" src="https://user-images.githubusercontent.com/360251/211671929-b0d7d5db-671a-4497-b44b-41475b8a7e5a.png">